src/NovelRT/Plugins/jsoncpp/jsoncpp.cpp

```diff
@@ -112,6 +112,16 @@ license you like.
  */
 
 namespace Json {
+
+//These defines are probably not needed for Windows, but seeing as Clang was unhappy I added them for MSVC too. :) - Ruby
+#ifdef __MSVC_VER
+#pragma warning( push )
+#pragma warning( disable : 4800 )
+#else
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunneeded-internal-declaration"
+#endif
```

Comment:
I didn't want to edit the jsoncpp vendored files, so I figured suppressing the warning may be better in this case. Its not our dependency to manage, and if it works then it works - no reason to try to bug fix dependencies we didn't write.